src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -1225,7 +1225,7 @@ Json::Value NetworkOPsImp::getOwnerInfo (
                     if (!jvObjects.isMember (jss::offers))
                         jvObjects[jss::offers] = Json::Value (Json::arrayValue);
 
-                    jvObjects[jss::offers].append (sleCur->getJson (0));
+                    jvObjects[jss::offers].append (sleCur->getJson (JsonOptions::none));
```

Comment:
I think the modified lines in this file need to be reformatted to fit within 80 columns.